test/vm.systemd-main

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -eu
+
+grep -q "platform:f*" /etc/os-release
+
+. /etc/os-release
+
+fedora_release="Fedora_${VERSION_ID}"
+
+echo "https://download.opensuse.org/repositories/system:systemd/${fedora_release}/system:systemd.repo"
+dnf config-manager addrepo --from-repofile="https://download.opensuse.org/repositories/system:systemd/${fedora_release}/system:systemd.repo"
+dnf -y update --setopt=install_weak_deps=False
```

Comment:
I suggest `--repo=systemd*` to make this more predictable

Reply:
Updated, the repository is actually called `system_systemd` a bit akward but works :)